build_tools/pkgci/external_test_suite/pytorch_models_cpu_llvm_task.json

```diff
@@ -16,10 +16,9 @@
   "expected_compile_failures": [
     // TODO(#17344): need to regenerate .mlirbc
     "opt-125M",
-    "resnet50",
-    // TODO(#17467): Remove the workaround once we have better support for attention op codegen.
-    "sdxl-vae-decode-tank"
+    "resnet50"
   ],
   "expected_run_failures": [
+    "sdxl-vae-decode-tank"
```

Comment:
Can we add a comment on why we are expecting this to fail?

Reply:
I think we have been carrying this for some time: https://github.com/iree-org/iree/pull/17620/files I only moved the failure further down again, from compile problems to runtime problems.

Reply:
It was put to this category when we created the file. I think we can leave what it is.. I can help with further debugging if it is a regression. It is not the case, so I think we don't block the PR because of this.

https://github.com/iree-org/iree/pull/17620/files#diff-48be6d52d0ac7aa92b98ad3065db9e644a9711a61e54b96b86f195d66ef7ec32